packages/paste-libraries/reakit/src/index.tsx

```diff
@@ -10,6 +11,7 @@ export {
   CompositeGroup,
   CompositeItem,
   // https://reakit.io/docs/disclosure/
+  useDisclosureContent,
```

Comment:
This file exposes these types to consuming applications, so unless we have an explicit need for consumers to use these new disclosure exports, we should avoid doing it here.

Reply:
Makes sense! I'll remove all the types that have been added here